adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboost.m

```diff
@@ -45,7 +44,7 @@ + (nonnull NSString *)adapterVersion {
 }
 
 + (nullable Class<GADAdNetworkExtras>)networkExtrasClass {
-  return [GADMChartboostExtras class];
```

Comment:
Can you clarify why the extras class is now removed?

Reply:
The information that was included in this class is not needed anymore.